src/renderer/components/+catalog/catalog.tsx

```diff
@@ -86,17 +88,6 @@ export class Catalog extends React.Component<Props> {
       reaction(() => this.entities, items => this.catalogEntityStore.loadItems(items), {
         fireImmediately: true,
       }),
-
-      // select initial category if nothing yet selected (via url-params)
-      when(() => this.categories.length > 0, () => {
```

Comment:
Default category for the catalog is `Browse All` if no additional params provided via URL. Wdyt?

Reply:
I think that is a good idea.